client/src/modules/invoices/registry/registry.js

```diff
@@ -117,38 +114,35 @@ function InvoiceRegistryController(
   // this function loads invoices from the database with search parameters
   // if passed in.
   function load(filters) {
-    var request;
-
     // flush error and loading states
     vm.hasError = false;
     toggleLoadingIndicator();
 
     // if we have search parameters and read invoices.
-    request = Invoices.read(null, filters);
+    const request = Invoices.read(null, filters);
 
     // hook the returned patients up to the grid.
     request
-      .then(function (invoices) {
-        invoices.forEach(function (invoice) {
+      .then(invoices => {
+        invoices.forEach(invoice => {
           invoice._backgroundColor = invoice.reversed ? reversedBackgroundColor : regularBackgroundColor;
-          invoice._is_cancelled = invoice.reversed;
         });
 
         // put data in the grid
         vm.uiGridOptions.data = invoices;
       })
       .catch(handler)
-      .finally(function () {
+      .finally(() => {
```

Comment:
is the inline syntax doesn't work ? 
`.finally(toggleLoadingIndicator);`

It is more simple and nice to see.

Reply:
Your suggestion will work fine here.  We can start doing it 👍